### ISSUE TITLE
fluentd config error: Unknown filter plugin 'sumo_fields'

### ISSUE THREAD

**Issue by rangedev** [SumoLogic/fluentd-kubernetes-sumologic#1]
FYI wrong plugin name. Should be "kubernetes_sumologic". 
```
2016-12-15 05:51:58 +0000 [info]: reading config file path="/fluentd/etc/fluent.conf"
2016-12-15 05:51:58 +0000 [info]: starting fluentd-0.12.31
2016-12-15 05:51:58 +0000 [info]: gem 'fluent-plugin-kubernetes_metadata_filter' version '0.26.2'
2016-12-15 05:51:58 +0000 [info]: gem 'fluent-plugin-record-reformer' version '0.8.2'
2016-12-15 05:51:58 +0000 [info]: gem 'fluent-plugin-sumologic_output' version '0.0.3'
2016-12-15 05:51:58 +0000 [info]: gem 'fluentd' version '0.12.31'
2016-12-15 05:51:58 +0000 [info]: adding match pattern="containers.**.fluentd**" type="null"
2016-12-15 05:51:58 +0000 [info]: adding filter pattern="containers.**" type="kubernetes_metadata"
2016-12-15 05:51:59 +0000 [info]: adding filter pattern="containers.**" type="kubernetes_sumologic"
2016-12-15 05:51:59 +0000 [info]: adding filter pattern="docker.**" type="sumo_fields"
2016-12-15 05:51:59 +0000 [error]: config error file="/fluentd/etc/fluent.conf" error="Unknown filter plugin 'sumo_fields'. Run 'gem search -rd fluent-plugin' to find plugins"
2016-12-15 05:51:59 +0000 [info]: process finished code=256
2016-12-15 05:51:59 +0000 [warn]: process died within 1 second. exit.
```

**Comment by stevezau** [comment]
@rangedev oops, must of missed that.

I'll fix shortly.

**Comment by nelsonenzo** [comment]
@stevezau Any update here?  Would like to get this working today if possible.

**Comment by stevezau** [comment]
@nelsonenzo I originally wrote a plugin to set the Sumo metadata fields for docker and kubernetes log files.

I intended to remove that plugin as there are generic plugins that can do the same thing. I was looking at the record transformer plugin. I've raised an issue trying to understand if it's possible to achieve what I need. https://github.com/sonots/fluent-plugin-record-reformer/issues/45

If i don't get a response i'll copy the plugin back in from the old repo
